### PR TITLE
docs: Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 3.4.0 - Feb 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Enhancement:
+- Improved local debug experience, more light weighted and more graceful.
+- Improved the Teams app authentication support.
+- Integrate Graph Toolkit with Teams Toolkit.
```
Comment (SLdragon) [inline packages/vscode-extension/CHANGELOG.md:9]: 3.4.0 do not contain Graph Toolkit related update, so you can remove this.

File: packages/vscode-extension/WHATISNEW.md [hunk 2]
```diff
@@ -8,6 +9,14 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.4.0 - Feb 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Enhancement:
+- Improved local debug experience, more light weighted and more graceful.
+- Improved the Teams app authentication support.
+- Integrate Graph Toolkit with Teams Toolkit.
```
Comment (SLdragon) [inline packages/vscode-extension/WHATISNEW.md:18]: Same here

File: packages/vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 3.4.0 - Feb 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Enhancement:
+- Improved local debug experience, more light weighted and more graceful.
+- Improved the Teams app authentication support.
```
Comment (blackchoey) [inline packages/vscode-extension/CHANGELOG.md:8]: We just fixed a bug in tab scaffolding template, we may remove this and see whether we need to tell users what bugs we fixed in the changelog.

File: packages/vscode-extension/WHATISNEW.md [hunk 4]
```diff
@@ -8,6 +9,14 @@
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
+## 3.4.0 - Feb 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Enhancement:
+- Improved local debug experience, more light weighted and more graceful.
+- Improved the Teams app authentication support.
```
Comment (blackchoey) [inline packages/vscode-extension/WHATISNEW.md:17]: same here

**Review by blackchoey — APPROVED** [review]

**Pull request merged**